style-guide/navigation-horizontal.html

```diff
@@ -0,0 +1,125 @@
+<html lang="en">
+<head>
+	<link rel="stylesheet" href="../src/css/framework.css" />
+	<link rel="stylesheet" href="../src/css/reset.css" />
+	<link rel="stylesheet" href="../src/css/typography.css" />
+	<link rel="stylesheet" href="../src/css/wsu-identity.css" />
+	<link rel="stylesheet" href="../src/css/wsu-web-identity.css" />
+	<link rel="stylesheet" href="../src/css/wsu-web-navigation.css" />
+</head>
+<body class="navigation-horizontal">
+	<header class="header-global">
+		The global header.
+	</header>
+	<nav class="navigation-main" role="navigation" aria-label="Site navigation">
+		<button id="js-navigation-toggle" class="navigation-toggle">Menu</button>
```

Comment:
Just for consideration - I've been adding `type="button"` to all buttons in the people directory. It looks redundant, but because that type has no default behavior, you don't have to add `e.preventDefault();` in the JavaScript.

There's also a `menu` type, which would be most appropriate... if it were implemented.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button.